scan.go

```diff
@@ -238,6 +239,16 @@ func scan(pkgPatterns []string) ([]*unit.SourceUnit, error) {
 		// compare results in tests more easily.
 		pkg.Root = ""
 
+		if strings.HasPrefix(pkg.ImportPath, "_") {
+			wd, err := os.Getwd()
+			if err != nil {
+				return nil, err
+			}
+			pkg.ImportPath = repo + pkg.ImportPath[1+len(wd):]
```

Comment:
Hmm, this approach is running into the problem that `go list -e -json ./...` does not follow symlinks. It can't scan the symlinked package directory inside of `.srclib-gopath`.